spec.bs

```diff
@@ -477,8 +483,9 @@ A [=component=] has an associated <dfn for=component>has regexp groups</dfn>, a
 </div>
 
 <div algorithm>
-  A {{URLPattern}} |pattern| <dfn export for=URLPattern>has regexp groups</dfn> if the following steps return true:
+  A {{URLPattern}} |urlpattern| <dfn export for=URLPattern>has regexp groups</dfn> if the following steps return true:
```

Comment:
Since this is useful to external specs, we should make it operate on [=URL pattern=]s, in a similar way to what you did for match.

Reply:
done.